### ISSUE TITLE
FileNotFoundError after installation

### ISSUE THREAD

**Issue by simonvh** [simonvh/genomepy#26]
```
$ genomepy install mm10 UCSC
downloading from http://hgdownload.soe.ucsc.edu/goldenPath/mm10/bigZips/chromFa.tar.gz...
done...
name: mm10
local name: mm10
fasta: /home/simon/.local/share/genomes/mm10/mm10.fa
Traceback (most recent call last):
  File "/home/simon/anaconda3/envs/gimme/bin/genomepy", line 11, in <module>
    load_entry_point('genomepy==0.5.1', 'console_scripts', 'genomepy')()
  File "/home/simon/anaconda3/envs/gimme/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/simon/anaconda3/envs/gimme/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/simon/anaconda3/envs/gimme/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/simon/anaconda3/envs/gimme/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/simon/anaconda3/envs/gimme/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/simon/anaconda3/envs/gimme/lib/python3.6/site-packages/genomepy/cli.py", line 38, in install
    regex=regex, invert_match=not(match), annotation=annotation)
  File "/home/simon/anaconda3/envs/gimme/lib/python3.6/site-packages/genomepy/functions.py", line 204, in install_genome
    generate_env()
  File "/home/simon/anaconda3/envs/gimme/lib/python3.6/site-packages/genomepy/functions.py", line 260, in generate_env
    for env in generate_exports():
  File "/home/simon/anaconda3/envs/gimme/lib/python3.6/site-packages/genomepy/functions.py", line 237, in generate_exports
    for name in list_installed_genomes():
  File "/home/simon/anaconda3/envs/gimme/lib/python3.6/site-packages/genomepy/functions.py", line 110, in list_installed_genomes
    return [f for f in os.listdir(genome_dir) if
FileNotFoundError: [Errno 2] No such file or directory: '~/.local/share/genomes/'
```

**Comment by simonvh** [comment]
Fixed with 0.5.4.

**Comment by Maarten-vd-Sande** [comment]
I just experienced this issue, after removing my installation and making a new config yaml. The genomepy.yaml has `~/.local/share/genomes` as genome dir. However to work with the tilde (~), you would need to expand this by os.path.expanduser.
